lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -123,6 +123,14 @@ public void shutdown() {
         shutdownAllEngines();
     }
 
+    public List<LocationListener> getListeners() {
```

Comment:
Since we decided to remove `LostApiClient#numberOfListeners()` can this also be removed?

I see its currently used in tests only. Can we switch back to using the `ShadowLocationManager` in those tests?